usr/lib/pkcs11/common/trace.h

```diff
@@ -121,42 +121,28 @@ extern struct trace_handle_t trace;
 void set_trace(struct trace_handle_t t);
 CK_RV trace_initialize();
 void trace_finalize();
-void ock_traceit(trace_level_t level, const char *fmt, ...)
-    __attribute__ ((format(printf, 2, 3)));
```

Comment:
Removing the __attribute__((format(printf, 2, 3))
disables all the format warnings - you will not do this.

Reply:
Fixed (reintroduced the attribute).